bin/metababel

```diff
@@ -4,6 +4,7 @@ require 'yaml'
 require 'optparse'
 require 'erb'
 require 'metababel'
+require 'fileutils'
 
 def validate_model(yaml)
   raise "Invalide Model #{yaml}" unless yaml[:stream_classes].all? do |d|
```

Comment:
Invalid ?

Reply:
Yes! Can you `add a suggestion`? :)  (https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request point 6, )